crates/router/src/connector/adyen/transformers.rs

```diff
@@ -874,218 +874,220 @@ pub enum OnlineBankingSlovakiaBanks {
     Viamo,
 }
 
-impl TryFrom<&api_enums::BankNames> for OnlineBankingSlovakiaBanks {
+impl TryFrom<&common_enums::enums::BankNames> for OnlineBankingSlovakiaBanks {
```

Comment:
why do we have `common_enums::enums` can't we directly use `common_enums` since there are re exports?